plugins/catalog-node/src/api/provider.ts

```diff
@@ -36,6 +36,11 @@ export interface EntityProviderConnection {
    * Applies either a full or delta update to the catalog engine.
    */
   applyMutation(mutation: EntityProviderMutation): Promise<void>;
+
+  /**
+   * Schedules a refresh on all of the entities that has a matching refresh key associated with the provided keys.
+   */
+  refresh(keys: string[]): Promise<void>;
```

Comment:
Let's make this `options: { keys: string[] }` so we can add other fields or variations of the argument in the future